README.md

```diff
@@ -98,28 +96,43 @@ pip install --upgrade pip
 pip install -e '.[all]' ## or 'all-gpu' for the GPU-enabled dependencies
 ```
 
-If you cannot upgrade `pip` to version 21.3 or higher, you will need to replace:
-- `'.[all]'` with `'.[sql,only-faiss,only-milvus,weaviate,graphdb,crawler,preprocessing,ocr,onnx,ray,dev]'`
-- `'.[all-gpu]'` with `'.[sql,only-faiss-gpu,only-milvus,weaviate,graphdb,crawler,preprocessing,ocr,onnx-gpu,ray,dev]'`
+**Custom Installation**
+You can choose the dependencies you want to install. To do so, specify them in the `pip install` command:
 
-For an complete list of the dependency groups available, have a look at the `haystack/pyproject.toml` file.
+```
+pip install 'farm-haystack[DEPENDENCY_OPTION]'
+```
+You can find a full list of dependency options at [haystack/pyproject.toml](https://github.com/deepset-ai/haystack/blob/main/pyproject.toml#L96).
+
+If you're running pip version earlier than 21.3, you can't install dependency groups that reference other groups. Instead, you can only specify groups that contain direct package references:
+
+```
+# instead of [all]
+pip install farm-haystack[sql,only-faiss,only-milvus1,weaviate,pinecone,opensearch,graphdb,inmemorygraph,crawler,preprocessing,ocr,onnx,ray,dev]
```

Comment:
A small one: wrap them up in quotes. Otherwise it can cause problems on zsh
'[all]'
'farm-haystack[sql,only-faiss,only-milvus1,weaviate,pinecone,opensearch,graphdb,inmemorygraph,crawler,preprocessing,ocr,onnx,ray,dev]'

same below, '[all-gpu]'
and 'farm-haystack[sql,only-faiss-gpu,only-milvus1,weaviatepinecone,opensearch,graphdb,inmemorygraph,crawler,preprocessing,ocr,onnx-gpu,ray,dev]'